### ISSUE TITLE
Could I use "Decode DICOM" component? 

### ISSUE THREAD

**Issue by mustafamerttunali** [doc-ai/tensorio-ios#208]
I'm looking for Dicom-Parser to load and view Dicom images in iOS using Swift, however, I haven't found dicom parser framework yet. It must be hard to parse DICOM image from zero to end. 

So, as we see in [here - Decode DICOM files for medical imaging](https://www.tensorflow.org/io/tutorials/dicom?hl=en), Could I use that component in tensorio-ios? If it is yes, how should I use it? 

If it is not, what should I do? Can I implement that using Swift by myself?

Thanks.

**Comment by phildow** [comment]
Hi @mustafamerttunali 

You won't be able to use TensorFlow IO code in Tensor/IO, so you won't be able to use that `tfio.image.decode_dicom_image` function. But if your model input expects some feature vector that's already been decoded from this format, as long as you can decode the dicom image you'll be able to pass that decoding into Tensor/IO.

It looks like you have GDCM and Imebra for options?

https://stackoverflow.com/questions/5278743/how-do-i-extract-pixel-data-from-dicom-files-on-ios
https://en.wikipedia.org/wiki/GDCM

https://imebra.com/

**Comment by mustafamerttunali** [comment]
I tried to send the DICOM file to the server, and processed it in there using some Javascript libraries and returned a vector in JSON but the code between server and iOS wasn't good, I couldn't make a good example. I was just trying to learn If I could process the DICOM files in iOS. 

I've look at these libraries but I was looking something different library such as Cornerstone.js.

Thanks for the information, have a nice day!
